lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -246,16 +259,17 @@ def self.upload(app_id, binary_path)
               UI.message("Uploading the APK/IPA")
               upload_binary(app_id, binary_path)
             end
-            status = upload_status(upload_token, app_id)
+            status, release_id = upload_status(upload_token, app_id)
           end
         end
+        release_id
       end
 
       def self.upload_status(app_token, app_id)
         response = connection.get("#{PATH}#{app_id}/upload_status/#{app_token}") do |request|
           request.headers["Authorization"] = "Bearer " + auth_token
         end
-        response.body[:status]
+        return response.body[:status], response.body[:release][:id]
```

Comment:
Clever! I'm not always a fan of returning tuples from methods, because it can make the interface kind of confusing. But in this case it's simple and cohesive enough that I like it.